client/notices.go

```diff
@@ -176,25 +196,27 @@ func (client *Client) WaitNotices(ctx context.Context, serverTimeout time.Durati
 	return jsonNoticesToNotices(jns), err
 }
 
-func makeNoticesQuery(opts *NoticesOptions) url.Values {
+func makeNoticesQuery(opts *NoticesOptions) (url.Values, error) {
```

Comment:
I think this always return a nil error now, so let's remove the error return again.